# Define the goal
action_server/Point2D target_point
action_server/Point2D start_point
---
# Define the result
action_server/Point2D[] waypoints
---
# Define a feedback message
bool state
